extensions-contrib/kafka-emitter/src/main/java/io/druid/emitter/kafka/KafkaEmitter.java

```diff
@@ -60,22 +62,28 @@
   private final ObjectMapper jsonMapper;
   private final MemoryBoundLinkedBlockingQueue<String> metricQueue;
   private final MemoryBoundLinkedBlockingQueue<String> alertQueue;
+  private final MemoryBoundLinkedBlockingQueue<String> requestQueue;
   private final ScheduledExecutorService scheduler;
 
   public KafkaEmitter(KafkaEmitterConfig config, ObjectMapper jsonMapper)
   {
     this.config = config;
+
     this.jsonMapper = jsonMapper;
+    this.jsonMapper.addMixIn(EmittingRequestLogger.RequestLogEvent.class, ClusterNameMixin.class);
```

Comment:
I would suggest to set this mixInAnnotation in `KafkaEmitterModule`. Please check `AvroExtensionsModule` as an example.